spec/features/audit/allocation/allocate_spec.rb

```diff
@@ -10,7 +10,7 @@
     second = create(:content_item, title: "content item 2")
     first = create(:content_item, title: "content item 3")
 
-    visit audits_allocations_path
+    visit audits_allocations_path(allocated_to: "anyone")
```

Comment:
I feel like we should be able to handle the absence of the `allocated_to` param and default to `anyone`.